pandas/tests/test_groupby.py

```diff
@@ -6465,6 +6465,14 @@ def testit(label_list, shape):
     testit(label_list, shape)
 
 
+def test_nunique_empty_Series():
+    series = Series(name='foo')
+    group = series.groupby(level=0)
+    result = group.nunique()
+    expected = Series(name='foo')
+    assert_series_equal(result, expected)
```

Comment:
Can you put this test inside the TestGroupBy class? Eg after `test_nunique_with_object` at Line 6404 (https://github.com/thanasis2028/pandas/blob/744e537c3c73ecb30a62a7707db138b7f4dafcd8/pandas/tests/test_groupby.py#L6400)